clean_build.sh

```diff
@@ -29,4 +35,8 @@ cd build || exit 1
 echo "Starting Barrier $B_BUILD_TYPE build..."
 $B_CMAKE $B_CMAKE_FLAGS .. || exit 1
 make || exit 1
+echo "INFO: Now commencing Barrier build process..."
+echo "INFO: We're building an $B_BUILD_TYPE output type."
+make || exit  1
```

Comment:
Somehow we now have two make lines

Reply:
OK, I don't know what that's happening, but once I push the final `clean_build.sh` change, we can take a further look.

Reply:
OK, I don't have that in my workspace, but the file in question on the GitHub web UI - looking at the latest rev of this PR, this isn't present. Is your tree outdated?